.pre-commit-hooks.yaml

```
@@ -0,0 +1,7 @@
+- id: verilator
```

```suggestion
---
- id: verilator
```

And please do a "make yamlfix" - only commit changes to this files not any others.

---

Done, https://github.com/verilator/verilator/pull/5452/commits/872cc8346cd2ebffe9be59afe580c47deb58f8ed

---

Note: we could use pre-commit in this repo, [yamlfix provides a corresponding hook](https://github.com/lyz-code/yamlfix/blob/1.17.0/.pre-commit-hooks.yaml). That makes integration of tools such as yamlfix into local workflows or CI trivial.

---

I think we'll get there, as there's already a hook for indenting the C++ and Python, but I don't trust the stability of yamlfix yet as using this is relatively new for me/us (based on numerous small pain points in getting C/Python indentation working).